### PR TITLE
Rename TEST_USERNAMES/TEST_UH_USERNAMES and TEST_NUMBERS/TEST_UH_NUMBERS to TEST_UIDS and TEST_UH_UUIDS

### PR DESCRIPTION
**Pull request by hokwaichan**
# Ticket Link

[1685](https://uhawaii.atlassian.net/browse/GROUPINGS-1685)

# List of squashed commits

- Rename TEST_USERNAMES/TEST_UH_USERNAMES and TEST_NUMBERS/TEST_UH_NUMBERS to TEST_UIDS and TEST_UH_UUIDS

# Test Checklist

- [x] Exhibits Clear Code Structure:
- [x] Project Unit Tests Passed:
- [x] Project Integration Tests Passed:
- [ ] Executes Expected Functionality:
- [ ] Tested For Incorrect/Unexpected Inputs:

### CODE REVIEW

File: src/test/java/edu/hawaii/its/api/service/TestGrouperApiService.java [hunk 1]
```diff
@@ -232,7 +232,7 @@ public void getSubjects() {
             assertTrue(testUhUuidsSet.contains(subject.getUhUuid()));
         }
 
-        // Numbers and Uids
+        // Uhuuids and Uids
         List<String> numbersAndUids = new ArrayList<>();
```
Comment (JorWo) [inline src/test/java/edu/hawaii/its/api/service/TestGrouperApiService.java:236]: Would be better to remove the comment then rename the `numbersAndUids` variable to `uhUuidsAndUids`.

File: src/main/resources/application-integrationTest.properties [hunk 2]
```diff
@@ -43,34 +43,34 @@ groupings.api.test.test-name-4=tst04name
...
+groupings.api.test.test-uh-uuid-5=99997056
+# test-dept-uh-uuid-1 left intentionally blank (See Project Security and Test Data in bwiki).
+groupings.api.test.test-dept-uh-uuid-1=
+groupings.api.test.test-dept-uh-uuid-2=testiwt2
+groupings.api.test.uh-uids=${groupings.api.test.test-uid-1},${groupings.api.test.test-uid-2},${groupings.api.test.test-uid-3},${groupings.api.test.test-uid-4},${groupings.api.test.test-uid-5}
+groupings.api.test.uh-uuids=${groupings.api.test.test-uh-uuid-1},${groupings.api.test.test-uh-uuid-2},${groupings.api.test.test-uh-uuid-3},${groupings.api.test.test-uh-uuid-4},${groupings.api.test.test-uh-uuid-5}
 groupings.api.test.names=${groupings.api.test.test-name-1},${groupings.api.test.test-name-2},${groupings.api.test.test-name-3},${groupings.api.test.test-name-4},${groupings.api.test.test-name-5}
-groupings.api.test.dept-uh-usernames=${groupings.api.test.test-dept-username-1},${groupings.api.test.test-dept-username-2}
-groupings.api.test.dept-uh-numbers=${groupings.api.test.test-dept-uh-number-1},${groupings.api.test.test-dept-uh-number-2}
+groupings.api.test.dept-uh-uids=${groupings.api.test.test-dept-uid-1},${groupings.api.test.test-dept-uid-2}
```
Comment (JorWo) [inline src/main/resources/application-integrationTest.properties:64]: Change `groupings.api.test.dept-uh-uids` to `groupings.api.test.dept-uids`.

File: src/main/resources/data.harness.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
-groupings.api.test.uh-usernames=testiwta,testiwtb,testiwtc,testiwtd,testiwte
-groupings.api.test.uh-numbers=99997010,99997027,99997033,99997043,99997056
+groupings.api.test.uh-uids=testiwta,testiwtb,testiwtc,testiwtd,testiwte
```
Comment (JorWo) [inline src/main/resources/data.harness.properties:1]: `groupings.api.test.uh-uids` should be `groupings.api.test.uids`

File: src/main/resources/application-integrationTest.properties [hunk 4]
```diff
@@ -43,34 +43,34 @@ groupings.api.test.test-name-4=tst04name
...
+groupings.api.test.test-dept-uid-2=testiwt2
+groupings.api.test.test-uh-uuid-1=99997010
+groupings.api.test.test-uh-uuid-2=99997027
+groupings.api.test.test-uh-uuid-3=99997033
+groupings.api.test.test-uh-uuid-4=99997043
+groupings.api.test.test-uh-uuid-5=99997056
+# test-dept-uh-uuid-1 left intentionally blank (See Project Security and Test Data in bwiki).
+groupings.api.test.test-dept-uh-uuid-1=
+groupings.api.test.test-dept-uh-uuid-2=testiwt2
+groupings.api.test.uh-uids=${groupings.api.test.test-uid-1},${groupings.api.test.test-uid-2},${groupings.api.test.test-uid-3},${groupings.api.test.test-uid-4},${groupings.api.test.test-uid-5}
```
Comment (JorWo) [inline src/main/resources/application-integrationTest.properties:61]: `groupings.api.test.uh-uids` should be `groupings.api.test.uids`

File: src/main/resources/application-localTest.properties [hunk 5]
```diff
@@ -21,14 +21,14 @@ groupings.api.test.grouping_single_include=${groupings.api.test.grouping_single}
...
-groupings.api.test.username=username
+groupings.api.test.uid=uid
 groupings.api.test.name=name
 groupings.api.test.uhuuid=uhuuid
 
 groupings.api.test.admin_user=test_admin_user
 groupings.api.grouping_owners=groupingOwners
-groupings.api.test.uh-usernames=testiwta,testiwtb,testiwtc,testiwtd,testiwte
-groupings.api.test.uh-numbers=99997010,99997027,99997033,99997043,99997056
+groupings.api.test.uh-uids=testiwta,testiwtb,testiwtc,testiwtd,testiwte
```
Comment (JorWo) [inline src/main/resources/application-localTest.properties:30]: `groupings.api.test.uh-uids` should be `groupings.api.test.uids`

File: src/test/java/edu/hawaii/its/api/groupings/GroupingAddResultTest.java [hunk 6]
```diff
@@ -26,11 +26,11 @@
 @SpringBootTest(classes = { SpringBootWebApplication.class })
 public class GroupingAddResultTest {
 
-    @Value("${groupings.api.test.uh-usernames}")
-    private List<String> TEST_USERNAMES;
+    @Value("${groupings.api.test.uh-uids}")
+    private List<String> TEST_UIDS;
```
Comment (JorWo) [inline src/test/java/edu/hawaii/its/api/groupings/GroupingAddResultTest.java:30]: Make sure to update all places with `groupings.api.test.uh-uids` to `groupings.api.test.uids`, thanks.

File: src/test/java/edu/hawaii/its/api/wrapper/RemoveMemberResultTest.java [hunk 7]
```diff
@@ -65,7 +65,7 @@ public List<String> getTestUids() {
         return new ArrayList<>(Arrays.asList(array));
     }
 
-    public List<String> getTestNumbers() {
+    public List<String> getTestUhuuids() {
```
Comment (JorWo) [inline src/test/java/edu/hawaii/its/api/wrapper/RemoveMemberResultTest.java:68]: This method should be named `getTestUhUuids()`, thanks.

File: src/test/java/edu/hawaii/its/api/wrapper/RemoveMembersResultsTest.java [hunk 8]
```diff
@@ -57,7 +57,7 @@ public List<String> getTestUids() {
         return new ArrayList<>(Arrays.asList(array));
     }
 
-    public List<String> getTestNumbers() {
+    public List<String> getTestUhuuids() {
```
Comment (JorWo) [inline src/test/java/edu/hawaii/its/api/wrapper/RemoveMembersResultsTest.java:60]: This method should also be named `getTestUhUuids()`.

File: src/test/resources/grouper.test.properties [hunk 9]
```diff
@@ -1,5 +1,5 @@
-groupings.api.test.uh-usernames=testiwta,testiwtb,testiwtc,testiwtd,testiwte
-groupings.api.test.uh-numbers=99997010,99997027,99997033,99997043,99997056
+groupings.api.test.uh-uids=testiwta,testiwtb,testiwtc,testiwtd,testiwte
```
Comment (JorWo) [inline src/test/resources/grouper.test.properties:1]: Make sure to update `groupings.api.test.uh-uids` to `groupings.api.test.uids` here too.

File: src/test/resources/ootb.grouper.test.properties [hunk 10]
```diff
@@ -1,5 +1,5 @@
-groupings.api.test.uh-usernames=testiwta,testiwtb,testiwtc,testiwtd,testiwte
-groupings.api.test.uh-numbers=99997010,99997027,99997033,99997043,99997056
+groupings.api.test.uh-uids=testiwta,testiwtb,testiwtc,testiwtd,testiwte
```
Comment (JorWo) [inline src/test/resources/ootb.grouper.test.properties:1]: And update `groupings.api.test.uh-uids` to `groupings.api.test.uids` here too.

**Comment by mhodgesatuh** [comment]
My only qualm is that the integration tests are checked off as running
successfully which I suspect is not true.  It appears that the team is
simply checking the boxes without fully comprehending what is being
communicated.  In this case, if the PR is deemed to be warranted because
the integration tests have been broken for a while, that needs to be called
out in the PR.

Michael Hodges (he/him/his)
Enterprise Middleware, Identity and Access Management
University of Hawaii, Information Technology Services
***@***.***
808-956-7195 (GMT -10:00)

we>$

**Review by JorWo — APPROVED** [review]
Looks good! Thanks for updating the variable names.

**Review by mhodgesatuh — APPROVED** [review]

**Pull request merged**